src/main/java/com/algomart/kibouregistry/controller/ParticipantsController.java

```diff
@@ -0,0 +1,67 @@
+package com.algomart.kibouregistry.controller;
+
+import com.algomart.kibouregistry.entity.Participants;
+import com.algomart.kibouregistry.exceptions.ResourceNotFoundException;
+import com.algomart.kibouregistry.exceptions.ResponseBuilder;
+import com.algomart.kibouregistry.services.ParticipantsService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/api/participants")
+public class ParticipantsController {
```

Comment:
Return proper response. You can have a Generic response for the purpose of this project. Refer to the endpoint documentation on postman for that